tests/sles4sap/trento/cluster_deploy.pm

```diff
@@ -0,0 +1,38 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Deploy SAP Landscape using qe-sap-deployment and network peering with Trento server
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment 'qesap_upload_logs';
+use base 'trento';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    $self->deploy_qesap();
+    my $cluster_rg = $self->get_qesap_resource_group();
+    my $cmd = '/root/test/00.050-trento_net_peering_tserver-sap_group.sh' .
```

Comment:
Just an option:
Got tip from Alvaro that you can use 'join()' here so it will handle the empty spaces within the command:
```
      my $cmd = join(" ", '/root/test/00.050-trento_net_peering_tserver-sap_group.sh',
          '-s', $self->get_resource_group,
          '-n $(az network vnet list -g ', $self->get_resource_group, '--query "[0].name" -o tsv)',
          "-t $cluster_rg",
          '-a $(az network vnet list -g', $cluster_rg, '--query "[0].name" -o tsv)');
```